internal/dev/registry.go

```diff
@@ -54,15 +54,15 @@ func SetupRegistry(ctx context.Context, image string, port int) error {
 		logger.Debugf("Reusing existing docker container %s on port %d for image registry", ftlRegistryName, port)
 	}
 
-	err = WaitForRegistryReady(ctx, port)
+	err = WaitForPortReady(ctx, port)
 	if err != nil {
 		return fmt.Errorf("registry container failed to be healthy: %w", err)
 	}
 
 	return nil
 }
 
-func WaitForRegistryReady(ctx context.Context, port int) error {
```

Comment:
The error messages in this function should be updated to remove references to "registry".